### PR TITLE
Maintenance

### PR DESCRIPTION
**Pull request by dianawu5**
fix bug in Maintenance filter
comment one useEffect in App.js

### CODE REVIEW

File: src/components/ViewBill.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    const handleModalCancel = () => {
+        setIsModalOpen(false);
+    };
+
+    return (
+        <>
+        <a type="link" onClick={showModal}>
+            View
+        </a>
+        <Modal title="View Request Details" open={isModalOpen} onCancel={handleModalCancel} footer={null}>
```
Comment (janicema1018) [inline src/components/ViewBill.js:20]: title="View Bill Details"

Comment (janicema1018) [inline src/components/ViewBill.js]: title="View Bill Details"

**Review by janicema1018 — CHANGES_REQUESTED** [review]

**Review by janicema1018 — COMMENTED** [review]
reviewed

**Review by janicema1018 — COMMENTED** [review]
code looks good

File: src/components/ViewBill.js (hunk 1 above)
Comment (dianawu5) [reply src/components/ViewBill.js]: changed

**Review by janicema1018 — APPROVED** [review]

**Pull request merged**